src/data/payments/data.ts

```diff
@@ -558,23 +558,8 @@ export interface IdealDetails {
 }
 
 export interface IngHomePayDetails {
```

Comment:
We debated changing any interfaces for deprecated payment methods to `any` - this would reflect Mollie's policy of completely removing the documentation pages for these methods.

Reply:
I'll leave the final call to you on whether you want to do this.

Reply:
I'm putting that on the todo list. I would like to experiment with the type of the `details` property being dependent on the value of `method`. Those two are related and should maybe be done in one go.